dotnet/src/IntegrationTests/TestSettings/OpenAIConfiguration.cs

```diff
@@ -10,12 +10,14 @@ internal sealed class OpenAIConfiguration
 {
     public string ServiceId { get; set; }
     public string ModelId { get; set; }
```

Comment:
```suggestion
```

Reply:
@RogerBarreto this change won't work because there is another settings for embeddings which uses `ModelId` and we use the same C# class for all the settings:
```json
"OpenAIEmbeddings": {
  "ServiceId": "text-embedding-ada-002",
  "ModelId": "text-embedding-ada-002",
  "ApiKey": ""
},
```

That's why I removed `Chat` prefix initially, reverted it back and proposed to keep it as it is for now as alternative, because settings and testing infrastructure need to be revisited completely.